account/identity.go

```diff
@@ -63,6 +106,8 @@ type IdentityRepository interface {
 	Save(ctx context.Context, identity *Identity) error
 	Delete(ctx context.Context, id uuid.UUID) error
 	Query(funcs ...func(*gorm.DB) *gorm.DB) ([]*Identity, error)
+	List(ctx context.Context) (*app.IdentityArray, error)
+	ValidIdentity(context.Context, uuid.UUID) bool
```

Comment:
should it be `ValidateIdentity` instead ? i.e., using a verb instead of a noun in the function name.